src/Microsoft.IdentityModel.Tokens/AsymmetricAdapter.cs

```diff
@@ -5,14 +5,6 @@
 using System.Security.Cryptography;
 using Microsoft.IdentityModel.Logging;
 
-#if NET45
-using System.Reflection;
-#endif
-
-#if NET461 || NET462 || NET472 || NETSTANDARD2_0 || NET6_0_OR_GREATER
```

Comment:
why removing this one?

Reply:
The X509Certificates is an unnecessary using.